coffee_core/src/coffee.rs

```diff
@@ -257,9 +264,10 @@ impl PluginManager for CoffeeManager {
     }
 
     async fn list(&mut self, remotes: bool) -> Result<CoffeeList, CoffeeError> {
+        self.remote_sync().await?;
```

Comment:
why not put this method inside the `inventory` method of the plugin manager?

Reply:
But then it will be called with coffee nurse itself and the error will be returned before the code runs

Reply:
Right, there is no way to check what kind of command the inventory is running?

Reply:
I searched and there was a way using an extra Boolean but this is cleaner in my opinion.

Reply:
lets keep this and think about a possible solution in the mean while